graphql/src/adapters/webserver/application.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from orca_shared.database.entities import PostgresConnectionInfo
+from orca_shared.database.use_cases import create_postgres_connection_uri
+from orca_shared.database.use_cases.validation import validate_config
+
+from src.adapters.graphql import initialized_adapters
+from src.adapters.graphql.initialized_adapters import adapters
+from src.adapters.graphql.initialized_adapters import logger_provider
+from src.adapters.logger_provider.basic import BasicLoggerProvider
+from src.adapters.storage.postgres import StorageAdapterPostgres
+from src.adapters.webserver.uvicorn_settings import INSTANTIATED_WEBSERVER_SETTINGS
+
+initialized_logger_provider = BasicLoggerProvider()
+initialized_adapters.logger_provider.static_logger_provider = initialized_logger_provider
+logger = initialized_adapters.logger_provider.static_logger_provider.get_logger("Setup")
```

Comment:
That would cause all logged statements to look the same. The point of this structure is to differentiate logging statements from different requests.

Reply:
Similar to the uuid string, I am having trouble seeing where this is a value add. Perhaps if I see a demo of logs so I better understand, it would be helpful.